quickstart-mongodbatlas/.taskcat.yml

```diff
@@ -0,0 +1,76 @@
+project:
+  name: quickstart-mongodb
+  owner: quickstart-eng@amazon.com
+  package_lambda: false
+  regions:
+  - ap-northeast-1
+  - ap-northeast-2
+  - ap-south-1
+  - ap-southeast-1
+  - ap-southeast-2
+  - ca-central-1
+  - eu-central-1
+  - eu-west-1
+  - eu-west-2
+  - sa-east-1
+  - us-east-1
+  - us-east-2
+  - us-west-1
+  - us-west-2
+  s3_bucket: ''
+tests:
+  mongodb-2-AZ:
+    parameters:
+      AvailabilityZones: $[alfred_genaz_2]
+      ClusterReplicaSetCount: '3'
+      KeyPairName: $[alfred_getkeypair]
+      MongoDBAdminPassword: T35TMong0
```

Comment:
Looks like a test suite for CFN?  If so even for test suite passwords we should use secrets or some random generated value.

Reply:
https://github.com/mongodb/mongodbatlas-cloudformation-resources/issues/52